execution_engine/src/core/engine_state/genesis.rs

```diff
@@ -57,7 +57,10 @@ impl GenesisResult {
             CommitResult::KeyNotFound(key) => GenesisResult::KeyNotFound(key),
             CommitResult::TypeMismatch(type_mismatch) => GenesisResult::TypeMismatch(type_mismatch),
             CommitResult::Serialization(error) => GenesisResult::Serialization(error),
-            CommitResult::Success { state_root, .. } => GenesisResult::Success {
+            CommitResult::Success {
+                state_root_hash: state_root,
+                ..
+            } => GenesisResult::Success {
                 post_state_hash: state_root,
```

Comment:
```suggestion
            CommitResult::Success {
                state_root_hash,
                ..
            } => GenesisResult::Success {
                post_state_hash: state_root_hash,
```

This applies to most changes within EE - there's no need to use `state_root_hash: state_root`.

Reply:
Done. I left out the places where it was referring to gRPC structs/methods.